app/aws-lsp-yaml-json-webworker/package.json

```diff
@@ -6,8 +6,8 @@
         "compile": "tsc --build",
         "clean": "rm -rf dist && rm -rf node_modules && rm -rf build",
         "build": "tsc",
-        "prepublishOnly": "npm run build && npm run compile && npm run bundle",
-        "bundle": "webpack",
+        "prepublishOnly": "npm run build && npm run compile && npm run package",
+        "package": "webpack",
```

Comment:
do we want to keep the `package` name in the scripts, why not rename all usages of it with something like `bundle` instead?

Reply:
is it because we still have a few subpackages that use `pkg` and do actual packaging?

Reply:
I wanted to keep naming consistent with older variants, I don't think it matters much and we used `package` for most of such commands